enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
+1. The user sets the `spec.decisionGroups[1].rolloutApproved` (`stage` decision group) value to `true`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The user sets the `spec.decisionGroups[2].rolloutApproved` (`prod` decision group) value to `true`.
+1. The updated policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `prod` group become compliant. Their `rolloutStatus` is set to `Succeeded`. The root policy
+   `status.rolloutStatus` value is `Succeeded`.
+
+#### Updated Policy With manualPerGroup Out of Order
+
+The below example is not realistic, but illustrates that if you have several production groups, you can choose whichever
+you'd like to be deployed to.
+
+1. The root policy is updated.
+1. All clusters have the `rolloutStatus` of `ToApply` and keep the previous policy version. The root policy
+   `status.rolloutStatus` value is `ToApply`.
+1. The user sets the `spec.decisionGroups[1].rolloutApproved` (`stage` decision group) value to `true`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`. The root policy `status.rolloutStatus` value is `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The user sets the `spec.decisionGroups[2].rolloutApproved` (`prod` decision group) value to `true`.
+1. The updated policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `prod` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The user sets the `spec.decisionGroups[0].rolloutApproved` (`stage` decision group) value to `true`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`. The root policy
+   `status.rolloutStatus` value is `Succeeded`.
+
+#### Common Situations
+
+##### A New Cluster Is Added During a Rollout
+
+When a new cluster is added, it receives the same policy as the others in the same group currently have. For example, if
+there is a policy update and a cluster is added to the `stage` group while the `dev` group is still `InProgress`, the
+new cluster would receive the last successful policy version, if any, until the rollout progressed to the `stage` group.
+
+If the cluster is added to an earlier group than the group that is `InProgress`, the rollout switches back to that group
+and waits for the cluster to rollout before resuming progress at the point previous to the new cluster being added.
```

Comment:
My understanding is. a newbie cluster is in progress as soon as it comes. And then the previously progressed cluster restarts its progress.. is this correct?

Reply:
Not quite. The new cluster always gets a policy. Either the last successful version or the latest version. Which one is dependent on if the cluster is in a group that is being rolled out to or has been rolled out to.

For example, if you have dev, stage, and prod groups and the rollout is on the stage group.

If the new cluster is a dev cluster, the rollout goes back to dev and applies the latest policy to the new cluster before going back to stage.

If the new cluster is a prod cluster, the new cluster gets the last successful policy until the rollout is on the prod group. Then the new cluster gets thte latest policy.